VstsTasks/AzureDtlCreateVM/task-funcs.ps1

```diff
@@ -105,12 +105,12 @@ function Get-DeploymentTargetResourceId
         [string] $DeploymentName,
         [string] $ResourceGroupName
     )
-    
+
     [Array] $operations = Get-AzureRmResourceGroupDeploymentOperation -DeploymentName $DeploymentName -ResourceGroupName $ResourceGroupName
 
     foreach ($op in $operations)
     {
-        if ($op.properties.targetResource -ne $null)
+        if ($null -ne $op.properties.targetResource)

```

Comment:
What if op.properties is null?

Reply:
The entire statement will be `null`. PS will not throw.